modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -144,6 +144,9 @@ TEST_P(Test_ONNX_layers, Convolution_variable_weight_bias)
          backend == DNN_BACKEND_INFERENCE_ENGINE_NN_BUILDER_2019) && target == DNN_TARGET_MYRIAD)
         applyTestTag(CV_TEST_TAG_DNN_SKIP_IE_MYRIAD, CV_TEST_TAG_DNN_SKIP_IE_NN_BUILDER, CV_TEST_TAG_DNN_SKIP_IE_NGRAPH);
 
+    if (backend == DNN_BACKEND_INFERENCE_ENGINE_NGRAPH && target == DNN_TARGET_CPU && getInferenceEngineCPUType() == "ARM")
```

Comment:
> `== "ARM"`

Please use macro defines instead of raw strings:
- it is very easy to make a typo in the string which is hard to find
- compilers raise error for typos in macro definition names

Reply:
Is it enough to add the macro 'CV_DNN_INFERENCE_ENGINE_CPU_TYPE_ARM_COMPUTE' or should we add a macro for CPU plugin too to use it in 'getInferenceEngineCPUType()`?
Do we need a public function 'getInferenceEngineCPUType()` or can we create a public `isArmComputePlugin()`?

Reply:
Similar to VPU part.

> isArmComputePlugin()

Lets keep it private. `getInferenceEngineCPUType()` is enough